src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/jira/StandaloneNodeFormula.kt

```diff
@@ -114,6 +115,39 @@ internal class StandaloneNodeFormula(
         connection: SshConnection,
         dbconfigXml: String
     ) {
+        if (!isMySql) {
+            connection.execute("sudo rm -f $dbconfigXml")
+
+            var dbconfigContent = """
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We can add the XML to `src/main/resources` and read it from classpath.
This way we'll be able to see XML with proper syntax highlight and verification.

Reply:
ok, I'll do this